Hamzah/temp.py

```diff
@@ -0,0 +1,22 @@
+# Convert a temperature of 100 degrees fahrenheit to celsius
+
+celsius_100 = (100 - 32)* 5/9
+print(celsius_100) 
+
+# Convert a temperature of 0 degrees fahrenheit to celsius
+
+celsius_0 = (0 - 32)* 5/9
+print(celsius_0)
+
+# Convert a temperature of 34.2 degrees fahrenheit to celsius
+
+print((34.2 - 32)* 5/9)
+
+# Convert a temperature of 5 degrees celsius to fahrenheit
+
+print((5 * 9/5)+ 32)
+
+# What is hotter, a temperature of 30.2 degrees celsius, or a temperature of 85.1 degrees fahrenheit?
+
+print((30.2 * 9/5)+ 32)
+print('30.2 degrees is hotter than 85.1')
```

Comment:
Thankfully we can just google this info now, but it was a great exercise in using code to convert it for us